go.mod

```diff
@@ -22,6 +22,7 @@ require (
 	github.com/juju/cmd/v3 v3.0.14
 	github.com/juju/collections v1.0.4
 	github.com/juju/errors v1.0.0
+	github.com/juju/names/v4 v4.0.0
```

Comment:
Why do we have direct use of names/v4 again? This was removed in a PR which landed yesterday.

Reply:
I think that we did not get rid of it completely. It looks like it is used in `application_test.go` `resource_model.go` still.